result/egressonlytracking.yaml

```diff
@@ -0,0 +1,290 @@
+        include_empty_metrics:
+          description: Controls inclusion/exclusion of tagged metrics where each underlying attributes has zero value or absent value.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        metric_names:
+          description: >-
+            The list of metric names that the returned result set will contain.
+            If the list is empty then all metrics will be returned.
+          type: array
+          items:
+            type: string
+            x-enum:
+              frames_tx:
+                x-field-uid: 1
+              frames_rx:
+                x-field-uid: 2
+              bytes_tx:
+                x-field-uid: 3
+              bytes_rx:
+                x-field-uid: 4
+              frames_tx_rate:
+                x-field-uid: 5
+              frames_rx_rate:
+                x-field-uid: 6
+              tx_l1_rate_bps:
+                x-field-uid: 7
+              rx_l1_rate_bps:
+                x-field-uid: 8
+              tx_rate_bytes:
+                x-field-uid: 9
+              rx_rate_bytes:
+                x-field-uid: 10
+              tx_rate_bps:
+                x-field-uid: 11
+              rx_rate_bps:
+                x-field-uid: 12
+              tx_rate_kbps:
+                x-field-uid: 13
+              rx_rate_kbps:
+                x-field-uid: 14
+              tx_rate_mbps:
+                x-field-uid: 15
+              rx_rate_mbps:
+                x-field-uid: 16
+          x-field-uid: 3
+        filters:
+          description: List of filters to selectively fetch tagged metrics with certain tag and corresponding value.
+          type: array
+          items:
+            $ref: '#/components/schemas/EgressOnlyTracking.MetricTag.Filter'
+          x-field-uid: 4
+    EgressOnlyTracking.MetricTag.Filter:
+      description: |-
+        A container for filtering ingress and/or egress metric tags.
+        The Tx stats may not be applicable in both the request and response filter.
+      type: object
+      properties:
+        name:
+          description: |-
+            A metric tag name that MUST exist in a packet or
+            egress_only_tracking configuration
+          type: string
+          x-field-uid: 1
+        values:
+          description: |-
+            A list of filters that can be applied to the metric tag name.
+            By default all values will be included in the egress-only-tracking metric results.
+          type: array
+          items:
+            type: string
+          x-field-uid: 2
+    EgressOnlyTracking.Metric:
+      description: |-
+        A container for egress-only-tracking metrics.
+        The container is keyed by the name and port_rx.
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of the egress-only-tracking flow
+          type: string
+          example: 'Tx -> Rx egress only'
+          x-field-uid: 1
+        port_tx:
```

Comment:
Need to understand if port_tx is really known. If yes, should clearly mention that this should be included only if implementation is able to detect if the received packet was transmitted by one of the ports in the configuration in set_config , otherwise it should not be included. Also, key fields should be one top. Not this.